### PR TITLE
previous next changes

### PR DESCRIPTION
**Pull request by mwernimont**
@ldecicco-USGS This adds the previous and next buttons onto the top of the blog pages just like the bottom of it. Added a CSS arrow in each button as requested. This is all revolving the single.html blog posts, which seems to be all what the blogs are using, but if you someday change your mind and not use the single.html the buttons will not keep all the code I gave it. Just a heads up!

### CODE REVIEW

**Comment by mwernimont** [comment]
@jiwalker-usgs Mind reviewing this? Think Laura's out all week

File: themes/hugo_theme_robust/layouts/_default/single.html [hunk 1]
```diff
@@ -37,11 +37,18 @@
...
 <div class="container">
-
+  
   <div class="row">
     <div class="col-md-9">
 
       <article class="single">
+        
+        {{ if .NextInSection }}
+<a href="{{.NextInSection.Permalink}}" class="previousPost"><div class="arrow-left"></div>Previous Post</a>
```
Comment (jiwalker-usgs) [inline themes/hugo_theme_robust/layouts/_default/single.html]: should these be `span`s instead of `div`s?

Comment (mwernimont) [inline themes/hugo_theme_robust/layouts/_default/single.html]: I suppose they could be. I don't really know the benefits or the cons

File: themes/hugo_theme_robust/static/css/styles.css [hunk 2]
```diff
@@ -600,7 +661,7 @@ article.single .body p>code {
   color: #78909c;
 }
 
-article.single .body h1 { margin: 2rem 0; font-weight: 700; }
+article.single .body h1 { margin: 2rem 0; font-weight: 700; clear:both; }
```
Comment (jiwalker-usgs) [inline themes/hugo_theme_robust/static/css/styles.css]: clear only for h1?  If this is a good idea it should probably be put on them all.

Comment (jiwalker-usgs) [inline themes/hugo_theme_robust/static/css/styles.css]: also, maybe throw in a space between : and both

**Comment by mwernimont** [comment]
@jiwalker-usgs Better?

**Comment by jiwalker-usgs** [comment]
:shipit:

**Pull request merged**